src/liballoc/boxed.rs

```diff
@@ -816,6 +816,24 @@ impl From<Box<str>> for Box<[u8]> {
     }
 }
 
+#[stable(feature = "box_from_array", since = "1.44.0")]
+impl<T, const N: usize> From<[T; N]> for Box<[T]>
+where
+    [T; N]: LengthAtMost32,
+{
+    /// Converts a `[T; N]` into a `Box<[T]>`
+    ///
+    /// This conversion allocates on the heap
+    /// and performs a copy of `slice`.
+    ///
+    /// # Examples
+    ///
+    /// FIXME
```

Comment:
I don't know that this needs examples, I think it's pretty self-explanatory.

Reply:
Okay, let me remove it.

Reply:
Oh no, I think we need examples, it should be like the other cases.